chart/values.yaml

```diff
@@ -2144,6 +2144,7 @@ pgbouncer:
 
   service:
     extraAnnotations: {}
+    ip: ~
```

Comment:
What do you think about `clusterIP` instead?

Reply:
good suggestion.